{
  "vulinfoid": "JVNDB-2020-006049",
  "vulinfodata": {
    "title": "rust-vmm vm-memory におけるバッファエラーの脆弱性",
    "vulinfodescription": {
      "overview": "rust-vmm vm-memory には、バッファエラーの脆弱性が存在します。"
    },
    "affected": {
      "affecteditem": [
        {
          "name": "vm-memory project",
          "productname": "vm-memory",
          "cpe": {
            "text": "cpe:/a:vm-memory_project:vm-memory",
            "version": "2.2"
          },
          "versionnumber": [
            "0.1.1 未満",
            "0.2.1 未満の 0.2.x"
          ]
        }
      ]
    },
    "impact": {
      "cvss": [
        {
          "version": "2.0",
          "severity": {
            "text": "Medium",
            "type": "Base"
          },
          "base": "5",
          "vector": "AV:N/AC:L/Au:N/C:N/I:N/A:P"
        },
        {
          "version": "3.0",
          "severity": {
            "text": "High",
            "type": "Base"
          },
          "base": "7.5",
          "vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
        }
      ],
      "impactitem": {
        "description": "サービス運用妨害 (DoS) 状態にされる可能性があります。"
      }
    },
    "solution": {
      "solutionitem": {
        "description": "ベンダより正式な対策が公開されています。ベンダ情報を参照して適切な対策を実施してください。"
      }
    },
    "related": {
      "relateditem": [
        {
          "type": "vendor",
          "name": "GitHub",
          "vulinfoid": "DoS issue when using virtio with rust-vmm/vm-memory #93",
          "url": "https://github.com/rust-vmm/vm-memory/issues/93"
        },
        {
          "type": "vendor",
          "name": "GitHub",
          "vulinfoid": "vm-memory-v0.1.1",
          "url": "https://github.com/rust-vmm/vm-memory/releases/tag/v0.1.1"
        },
        {
          "type": "vendor",
          "name": "GitHub",
          "vulinfoid": "vm-memory-v0.2.1",
          "url": "https://github.com/rust-vmm/vm-memory/releases/tag/v0.2.1"
        },
        {
          "type": "advisory",
          "name": "Common Vulnerabilities and Exposures (CVE)",
          "vulinfoid": "CVE-2020-13759",
          "url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-13759"
        },
        {
          "type": "advisory",
          "name": "National Vulnerability Database (NVD)",
          "vulinfoid": "CVE-2020-13759",
          "url": "https://nvd.nist.gov/vuln/detail/CVE-2020-13759"
        },
        {
          "type": "cwe",
          "name": "JVNDB",
          "vulinfoid": "CWE-119",
          "url": "https://jvndb.jvn.jp/ja/cwe/CWE-119.html",
          "title": "バッファエラー"
        }
      ]
    },
    "history": {
      "historyitem": [
        {
          "historyno": "1",
          "datetime": "2020-06-29T16:12:47+09:00",
          "description": "[2020年06月29日]\\n  掲載"
        }
      ]
    },
    "datefirstpublished": "2020-06-29T16:12:47+09:00",
    "datelastupdated": "2020-06-29T16:12:47+09:00",
    "datepublic": "2020-05-28T00:00:00+09:00"
  }
}
